src/main/scala/org/broadinstitute/dsde/firecloud/model/DataUse.scala

```diff
@@ -26,6 +27,10 @@ object DataUse {
   case class DiseaseOntologyNodeId(uri: Uri, numericId: Int)
   object DiseaseOntologyNodeId {
     def apply(stringid:String) = {
+      println("id " + stringid)
+      println("pre " + doid_prefix)
+      println("result " + stringid.startsWith(doid_prefix))
```

Comment:
`println`

Reply:
Still doing some work on this, will let you know when it's back in reviewable state!